netsim/cli/validate.py

```diff
@@ -498,7 +498,7 @@ def get_parsed_result(v_entry: Box, n_name: str, topology: Box, verbosity: int)
   # Set up arguments for the 'netlab connect' command and execute it
   #
   args = argparse.Namespace(quiet=True,host=n_name,output=True,show=v_cmd,verbose=False)
-  result = connect_to_node(args=args,rest=[],topology=topology,log_level=LogLevel.NONE)
+  result = connect_to_node(args=args,rest=[],topology=topology,node=n_name,log_level=LogLevel.NONE)
```

Comment:
Just put node to the first place here because it's the most important argument ;)